Assembler.UnitTests/FrameHandlers/MiddleFrameHandlerTests.cs

```diff
@@ -0,0 +1,138 @@
+﻿using System;
+using Assembler.Base.FrameHandlers;
+using Assembler.Core;
+using Assembler.Core.Entities;
+using Assembler.Core.Enums;
+using Moq;
+using NUnit.Framework;
+
+namespace Assembler.UnitTests.FrameHandlers
+{
+    [TestFixture]
+    public class MiddleFrameHandlerTests
```

Comment:
See comment of InitialFrameHandlerTests and apply here

Reply:
f

Reply:
f